app/views/components/docs/contents-list.yml

```diff
@@ -38,6 +38,17 @@ examples:
           text: Another pretty long contents list entry, not as long as the first, but still a little.
         - href: "#third-thing"
           text: Third thing
+  blank_href:
```

Comment:
I'd call this example 'Contents items without links' or 'Text only content items'
"Blank href" is not clear.